.travis.yml

```diff
@@ -1,18 +1,31 @@
-
 language: clojure
 jdk:
-  - oraclejdk8
-# check that it compiles && run linters && run tests
-# stop if any of them fails
-#script: lein do clean, compile, check, eastwood '{:exclude-namespaces [hiposfer.service.conjectures.specs]}', test
+- oraclejdk8
 script:
-  - lein clean
-  - lein compile
-  - lein check
-  - lein eastwood '{:exclude-namespaces [hiposfer.kamal.graph.specs hiposfer.kamal.spec]}'
-  - lein test
-#TODO: slack notification
+- lein clean
+- lein compile
+- lein check
+- lein eastwood '{:exclude-namespaces [hiposfer.kamal.graph.specs hiposfer.kamal.spec]}'
+- lein test
 notifications:
   email:
-    on_success: change # default: change
-    on_failure: always # default: always
+    on_success: change
+    on_failure: always
+env:
+  global:
+  - secure: piWUUgJePy5EFQmVyd/dAN4EG9wgubzr+i5qfmcyjbuhLPd+DzI1SYIhGDJ0PcrZDPVf/vG0uCCyWbFXaNCB4LJETt1/N7stR5H7mGl8QY05KBB04DFlfqUf3gfIeQCfILfxdpI1+//5YNUQHVgXB0RQ25FreHEOpPvS02uOziNVifrwQgMH2zYgMVycUSqsb1fkU+/obTU8DeeGrdexHLPhacRYJr8UYury98uZHVFfpdd6zFp7lTqGf2js424u/ER7zNmrUQiVzIEtNp86ZOh+IeUUGPETzf3OJhDntPpHlbDzE09DkJt16qGo5HUZjtH//+kO3nBoZid5SA1jY6Lk1UI9Piw3Ihnob5XwCZ4xiI3ts6vfeOU1HFOvtAoVe7ZDoyF/ErEZlvLM+VTvbhhQjJuNJq1hqc2XJbcg7IrvmDYJbdcI4x3MBt3HD/slUTlm2wZB0XQCpuYMWDrT99otTfKzM996tWa9h1mNalcSooLOMaKhuQxy448K14wI2rWObr8xvK6swMbuk5ZUsi+HiFwNcMORIjUvCUbbyq2ZVPfVYhHQB7SzwNtsZhXtpOJ7CNSIrrCUV8Y65xOdk9by9VA635GC1FRysrm/yj1S5V+sCXqjI2rO2s0aUCC7hh/EMqdLEkyTbbE76xczF2ypNl7mdL6bECAttLdHzGs=
+  - secure: aj5xYi4auQ2fInEVfA8NrbNqz+0DwBhWNa6jtrd7/+jtEgxyUpFe3N+KnSVfyDXiPdFlv2u2Z+IO0LzG79D0Db0LdhHII83UXg1x296AC+ydZwSXmmc4BeK1fSYYkc56JMhyW3ovPsfdSCnqS2eutXuqOj49QnjHq097QqZZPj9Izoghwzwnv9L6xoeTmPAW8M6vDjhxZULArW1gwCEILGGm+sLlxksPxY8WH226kHSk4c8FiORTYjAIaoIHzxhamkH9+kc0AMCHZz7/48T31Gs3xA6ZWdh1eXikSM3nwM5BusQZq6xwFEJMpP96/nhERIk+YpsbMCxlWsHNpMtBA5A2wh3RTEqNdpVi3J1F5PkDrnIIjb7jAOJDyOlY+9/WGclDqoXT4YsdoDm9a6z7btdna9Mo+vAfoCKlqDWryyjLJQfQlCYwaH+l2e0Qp4uU7XxSm2m6AaMitxzFZmuh4cEME1CR5syquVnI/Qr51CbY+ZIsjmqsPCk5587ig9sNNH5FUwDVv6R0tDsb9JkFmtfheQp3Bxml+y83NkJnOg+oUPkrZW4eZVROgpoDJHp5+WJBlIeiwX5shEMPeoAoPdjCNQoIAILDIjQV84HUwW9zn9TMdPFESW+31kPUARy8AfWd8LS5yzRmRKe/Xf5Jxg0f7Bc2h9mnTptTltmydSY=
+deploy:
+- provider: script
+  skip_cleanup: true
+  script: lein deploy
+  on:
+    branch: master
+- provider: releases
```

Comment:
@mehdisadeghi I guess that this is going to github.com right? nevertheless it was not mentioned in the PR :(

Reply:
@carocad yes true, I pushed both changes in one PR, sorry for the confusion! It is supposed to create a release when a new tag is found. We have to test it however :zap: